op-bindings/predeploys/addresses.go

```diff
@@ -139,7 +139,7 @@ func init() {
 	Predeploys["UniswapV3Factory"] = &Predeploy{Address: UniswapV3FactoryAddr, ProxyDisabled: true}
 	Predeploys["NFTDescriptor"] = &Predeploy{Address: NFTDescriptorAddr, ProxyDisabled: true}
 	Predeploys["NonfungiblePositionManager"] = &Predeploy{Address: NonfungiblePositionManagerAddr, ProxyDisabled: true}
-	Predeploys["NonfungibleTokenPositionDescriptor"] = &Predeploy{Address: NonfungibleTokenPositionDescriptorAddr}
+	Predeploys["NonfungibleTokenPositionDescriptor"] = &Predeploy{Address: NonfungibleTokenPositionDescriptorAddr, ProxyDisabled: true}
```

Comment:
@AaronLee22 
Can you check this code? 🙇🏻‍♂️

Reply:
I read the code and guess `NonfungibleTokenPositionDescriptor` isn't the proxy contract. Let's wait for Aaron to confirm it.

Reply:
I remember that NonfungibleTokenPositionDescriptor.sol is the implementation contract for TransparentUpgradeableProxy.sol contract, for some reason by contract team. Let me ask one more time and tell you by next Mon.

Reply:
https://etherscan.io/address/0xEe6A57eC80ea46401049E92587E52f5Ec1c24785#readProxyContract
if you take a look into this address(TransparentUpgradable Proxy contract) -> This is the Proxy contract

0x91ae842A5Ffd8d12023116943e72A606179294f3 -> NonfungibleTokenPositionDescriptor is used as implementation contract.

Reply:
Same thing here.
https://optimistic.etherscan.io/address/0xEe6A57eC80ea46401049E92587E52f5Ec1c24785#code -> Official optimism address for UniswapV3

Reply:
@AaronLee22 That mean we need to config that `NonfungibleTokenPositionDescriptor` with `ProxyDisabled` is `true`, right?

Reply:
No, NonfungibleTokenPositionDescriptor has to be set as 

Predeploys["NonfungibleTokenPositionDescriptor"] = &Predeploy{Address: NonfungibleTokenPositionDescriptorAddr}

not ProxyDisabled: true, -> this mean you will disable the proxy pattern.

In this case we have to use proxy pattern.

Reply:
> No, NonfungibleTokenPositionDescriptor has to be set as
> 
> Predeploys["NonfungibleTokenPositionDescriptor"] = &Predeploy{Address: NonfungibleTokenPositionDescriptorAddr}
> 
> not ProxyDisabled: true, -> this mean you will disable the proxy pattern.
> 
> In this case we have to use proxy pattern.

Hmm I got it, So we are missing something with the config. Let me check. Sorry you because my lacking knowledge about it